src/scripts/components/VideoForm.js

```diff
@@ -204,7 +150,12 @@ class VideoForm extends Component<Props, Object> {
   }
 
   publishVideo (publish: false) {
-    this.saveData(publish)
+    this.props.openModal({
+      modal: 'ModalStake',
+      callback: () => {
```

Comment:
Can `callback` be named to be more descriptive? When is this called? Is it when the modal opens? Closes? Can this possibly be updated so that we have the option of passing in an `onOpen` and `onClose` callback?

Reply:
@bent0b0x this _callback_ is more generic, used whenever it needs. In this case, it used when the submit is success on ModalStake

Reply:
Hm I'm not sure about that then. It is really unclear what `callback` does. Is this specific to `ModalStake`? If so shouldn't it be handled by `ModalStake` itself?